modular_skyrat/modules/primitive_fun/code/big_mortar.dm

```diff
@@ -0,0 +1,115 @@
+/obj/structure/large_mortar
+	name = "large mortar"
+	desc = "A large bowl perfect for grinding and or juicing a large number of things at once."
+	icon = 'modular_skyrat/modules/primitive_fun/icons/cooking_structures.dmi'
+	icon_state = "big_mortar"
+	density = TRUE
+	anchored = TRUE
+	max_integrity = 100
+	pass_flags = PASSTABLE
+	custom_materials = list(/datum/material/wood = MINERAL_MATERIAL_AMOUNT * 10)
+	/// All of the items stored within this structure
+	var/list/contained_items
+	/// The maximum number of items this structure can store
+	var/maximum_contained_items = 5
+
+/obj/structure/large_mortar/Initialize(mapload)
+	. = ..()
+	create_reagents(200, OPENCONTAINER)
+
+/obj/structure/large_mortar/examine(mob/user)
+	. = ..()
+	. += span_notice("It currently contains <b>[LAZYLEN(contained_items)]/[maximum_contained_items] items.")
+	. += span_notice("It can be unsecured from the ground with <b>Right Click</b>")
+	. += span_notice("You can empty all of the items out of it with <b>Alt Click</b>")
+
+/obj/structure/large_mortar/Destroy()
+	drop_everything_contained()
+	return ..()
+
+/obj/structure/large_mortar/AltClick(mob/user)
+	if(!LAZYLEN(contained_items))
+		balloon_alert(user, "nothing inside")
+		return
+	drop_everything_contained()
+	balloon_alert(user, "removed all items")
+
+/obj/structure/large_mortar/proc/drop_everything_contained()
+	if(!LAZYLEN(contained_items))
+		return
+	for(var/obj/target_item in contained_items)
+		target_item.forceMove(drop_location())
+		LAZYREMOVE(contained_items, target_item)
+
+/obj/structure/large_mortar/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+		return
+	if(!can_interact(user) || !user.canUseTopic(src, be_close = TRUE))
+		return
+	set_anchored(!anchored)
+	balloon_alert_to_viewers("unsecured")
+	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+/obj/structure/large_mortar/attackby(obj/item/attacking_item, mob/living/carbon/human/user)
+	..()
```

Comment:
should be . = ..()

But really, this should be at the end and be a return ..()